package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "banidb-api",
-  "version": "2.0.0-beta.7",
+  "version": "2.0.1",
```

Comment:
is this just bug fixes? typically semvar's `major.minor.patch` follows the pattern, major release is breaking change, minor is new features but usually backwards compatible, and patch is bug fixes. https://semver.org/